src/main/java/com/fakng/fakngagrgtr/parser/amazon/AmazonParser.java

```diff
@@ -18,23 +18,24 @@
 import com.fakng.fakngagrgtr.persistent.vacancy.Vacancy;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
-import org.springframework.web.reactive.function.client.WebClient;
 import org.springframework.web.reactive.function.client.WebClient.ResponseSpec;
 
 @Component
 public class AmazonParser extends ApiParser {
 
     // Amazon doesn't allow to get more than 10000 vacancies.
     private static final int MAX_VACANCIES_COUNT = 10000;
+
+    @Value("${amazon.result_limit:100}")
```

Comment:
Do I understand correctly that we use this address to create the number 100 and embed it in resultLimit? Is there no such parameter in application.yml? If so, why is it so difficult? We have a constructor in which we can assign some RESULT_LIMIT there.